src/Core/Resolvers/QueryExecutor.cs

```diff
@@ -23,6 +25,8 @@ namespace Azure.DataApiBuilder.Core.Resolvers
     public class QueryExecutor<TConnection> : IQueryExecutor
         where TConnection : DbConnection, new()
     {
+        private const string TOTALDBXEXECUTIONTIME = "TotalDbExecutionTime";
```

Comment:
Nit: Did you mean to name this variable `TOTALDBEXECUTIONTIME`? Adding underscore between each word may help.

Reply:
yes let me fix.